terraform/datadog_dashboards.tf

```diff
@@ -51,15 +51,15 @@ resource "datadog_dashboard" "ordered_dashboard" {
             query {
               metric_query {
                   data_source = "metrics"
-                  query       = "avg:sonar_cloud.measures.lines_to_cover{org:twilio,pre-release:false} by {repo}"
+                  query       = "avg:sonar_cloud.measures.Metrics.LINES_TO_COVER{org:twilio,pre-release:false} by {repo}"
```

Comment:
This issue started on [this run](https://github.com/sendgrid/dx-automator/actions/runs/3394380380/jobs/5642868964) (compared with the [previous run](https://github.com/sendgrid/dx-automator/actions/runs/3346049480/jobs/5542362765)) where the version of python flipped to 3.11

We should lock down python to a specific version using a `.python-version` file as recommended [here](https://github.com/sendgrid/dx-automator/actions/runs/3346049480/jobs/5542362765) and should see why these tests for the metric collection aren't failing.

Reply:
Yeah I can pin the 3.10 version. Wondering if using the [python-version](https://github.com/actions/setup-python/blob/main/action.yml#L6) input is a better idea or creating the .python-version file? 

The tests also started failing on [3.11](https://github.com/sendgrid/dx-automator/actions/runs/3579623000/jobs/6021062729#step:5:10) vs compared to the previous [3.10](https://github.com/sendgrid/dx-automator/actions/runs/3372055459/jobs/5595035917#step:5:10) one.

Reply:
I like the file version as it's 1 place to set it which can be used locally as well as by all of the GH workflows.